src/praisonai/praisonai/gateway/kanban_dispatcher.py

```
@@ -90,19 +109,36 @@ async def dispatch_once(self) -> int:
                     logger.debug(f"Failed to claim task {task.id} (already claimed)")
                     continue
                 
+                # Open a run (attempt) for this claim so failures/retries
+                # have a durable, structured record.
+                run_id = None
+                try:
+                    run_id = store.start_run(task.id, profile=self.worker_id)
+                except Exception as run_err:
```

Ruff (0.15.18)

[warning] Do not catch blind exception: `Exception`

(BLE001)